bin/start-seatunnel.sh

```diff
@@ -146,6 +146,8 @@ if [ ! -z "${variables_substitution}" ]; then
   clientModeDriverJavaOpts="${variables_substitution}"
 fi
 
+driverJavaOpts="${driverJavaOpts} -Dlog4j.configuration=file:${CONF_DIR}/log4j.properties"
+executorJavaOpts="${executorJavaOpts} -Dlog4j.configuration=file:${CONF_DIR}/log4j.properties"
 
```

Comment:
@RickyHuo This seems not reasonable, if users specify their log properties wherever they want in seatunnel file, it will not works.The default log4.properties is set in ExposeSparkConf.java and can be override if uses have set.

Reply:
try to simplify start-seatunnel.sh and update codes of ExposeSparkConf.java, for code readability, if possible